src/commands.rs

```diff
@@ -3950,53 +3955,81 @@ fn is_fast_forward(repo: RepoRef, branch_name: &str, new_target_id: &CommitId) -
 
 fn cmd_branch(ui: &mut Ui, command: &CommandHelper, args: &BranchArgs) -> Result<(), CommandError> {
     let mut workspace_command = command.workspace_helper(ui)?;
-    let branch_name = &args.name;
+    let branch_names: Vec<&str> = if args.delete || args.forget {
+        args.names
+            .iter()
+            .map(|branch_name| {
+                match workspace_command
+                    .repo()
+                    .view()
+                    .get_local_branch(branch_name)
+                {
+                    Some(_) => Ok(branch_name.as_str()),
+                    None => Err(CommandError::UserError(format!(
+                        "No such branch: {}",
+                        branch_name
+                    ))),
+                }
+            })
+            .try_collect()?
+    } else {
+        args.names.iter().map(|name| name.as_str()).collect()
+    };
+
+    if branch_names.is_empty() {
+        ui.write_warn("warn: no branches provided\n")?;
```

Comment:
I think we decided to capitalize the first word. Also period at the end? There's another other similar message further down to update.

Also, what do you think about "warn" vs "warning"? I've used "Warning" in a single place so far, so it's easy to change if we prefer "warn". I prefer "Warning" because it's a noun like "Error", "Note", and "Info".

Reply:
> I think we decided to capitalize the first word.

Capitalize "warning", or capitalize the first word after the colon, "no"?

> Also, what do you think about "warn" vs "warning"?

I was using "warn" since it's shorter and it's the typical log-level term (e.g. https://docs.rs/log/latest/log/enum.Level.html#variant.Warn), but I don't feel strongly about it.

Reply:
> > I think we decided to capitalize the first word.
> 
> Capitalize "warning", or capitalize the first word after the colon, "no"?

Good question. I'd say both. I feel like the part starting with "no" is the actual message and the presence of a "Warning:" label shouldn't affect it. But that's just my feeling.

> 
> > Also, what do you think about "warn" vs "warning"?
> 
> I was using "warn" since it's shorter and it's the typical log-level term (e.g. https://docs.rs/log/latest/log/enum.Level.html#variant.Warn), but I don't feel strongly about it.

I see. In the user-facing messages I could find quickly by grepping for "warn:" and "warning:" in the hg and git source repos, it seems they all use "warning".